docs/source/user/api.rst

```diff
@@ -15,9 +15,33 @@ consumed by FreeRADIUS.
 Organization API Token
 ----------------------
 
-Only requests containing the right API token and Organization UUID will able
-to talk to the API endpoints consumed by freeradius
-(`Authorize`_, `Post Auth`_, `Accounting`_).
+There are three ways to authenticate your requests:
+
+1. Radius token (recommended)
+2. Bearer token
+3. Querystring
```

Comment:
Are you say this line should be added as a short description here? Because the information is mentioned below.
I am changing the order to make things more clear.

Reply:
Completely changed the order in order to make it more clear, please let me know if it needs any further change.